packages/puppeteer-extra-plugin-stealth/evasions/navigator.hardwareConcurrency/index.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const {PuppeteerExtraPlugin} = require('puppeteer-extra-plugin')
+
+/**
+ * Allows faking CPU core number. This allows creating different profiles.
+ * Will overwrite `navigator.hardwareConcurrency` property.
+ */
+class Plugin extends PuppeteerExtraPlugin {
+    constructor(opts = {}) {
+        super(opts)
```

Comment:
There's some linting issues in the file (using 4 spaces instead of 2). Quick way to fix those:

```bash
npx prettier@1.19.1 --write evasions/navigator.hardwareConcurrency/index.js
```